### PR TITLE
refresh csv grid files

### LINKED ISSUE #216: `vn_simona` in SIMONA seems to uses deprecated csv file version
New csv geo json:
"{""type"":""Point"",""coordinates"":[7.411111,51.492528],""crs"":{""type"":""name"",""properties"":{""name"":""EPSG:4326""}}}"

Old csv geo json:
"{"type":"Point","coordinates":[7.4116482,51.4843281],"crs":{"type":"name","properties":{"name":"EPSG:0"}}}"

Reading files with the old format which is not specification XYZ compliant leads to problems when used in other grids.
This sould be updated. 

One could probably just read it in with the current PSDM implementation and persist it again to get rid of this problem.

### PR DESCRIPTION
**Pull request by t-ober**
Resolves #216

### CODE REVIEW

**Comment by t-ober** [comment]
Conversion was done with [this](https://github.com/ie3-institute/PowerSystemDataModel/pull/583) adaption

File: input/samples/vn_simona/fullGrid/wec_type_input.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
+uuid,capex,cos_phi_rated,cp_characteristic,eta_conv,hub_height,id,opex,rotor_area,s_rated
+b70374f3-1feb-45af-8011-1304a532d398,0.0,0.9,"cP:{(0.00,0.00),(1.00,0.00),(2.00,0.12),(3.00,0.29),(4.00,0.40),(5.00,0.43),(6.00,0.46),(7.00,0.48),(8.00,0.49),(9.00,0.50),(10.00,0.49),(11.00,0.42),(12.00,0.35),(13.00,0.29),(14.00,0.23),(15.00,0.19),(16.00,0.15),(17.00,0.13),(18.00,0.11),(19.00,0.09),(20.00,0.08),(21.00,0.07),(22.00,0.06),(23.00,0.05),(24.00,0.05),(25.00,0.04),(26.00,0.04),(27.00,0.03),(28.00,0.02),(29.00,0.01),(30.00,0.01),(31.00,0.00),(32.00,0.00),(33.00,0.00),(34.00,0.00),(35.00,0.00),(36.00,0.00),(37.00,0.00),(38.00,0.00),(39.00,0.00),(40.00,0.00),(41.00,0.00),(42.00,0.00),(43.00,0.00),(44.00,0.00),(45.00,0.00),(46.00,0.00),(47.00,0.00),(48.00,0.00),(49.00,0.00),(50.00,0.00)}",100.0,98.0,EnerconE-82,0.0,5281.02,2222.2
```
Comment (t-ober) [inline input/samples/vn_simona/fullGrid/wec_type_input.csv:2]: It seems like here was some considerable rounding done .. find out why that is

**Comment by t-ober** [comment]
Also: Why is there this big discrepancy between lines removed and lines added ?

File: input/samples/vn_simona/fullGrid/wec_type_input.csv (hunk 1 above)
Comment (sebastian-peter) [reply input/samples/vn_simona/fullGrid/wec_type_input.csv:2]: Depending on https://github.com/ie3-institute/PowerSystemDataModel/issues/710

**Comment by jo-bao** [comment]
@t-ober I think #424 solved #216 and the problem here (roundings). So can this PR be closed? Or is there something else to be done?

**Comment by sebastian-peter** [comment]
Resolved with #424

**Pull request closed without merging**